src/Auth.cpp

```diff
@@ -90,6 +90,7 @@ DeviceCertToTokenFunc CredentialProvider::deviceCertToTokenCallback() {
 
 STATUS CredentialProvider::getStreamingTokenHandler(UINT64 custom_data, PCHAR stream_name, STREAM_ACCESS_MODE access_mode, PServiceCallContext p_service_call_context) {
     LOG_DEBUG("getStreamingTokenHandler invoked");
+    STATUS status = STATUS_SUCCESS;
```

Comment:
The initial value doesn't matter since it will get overwritten on L115 and L120. There is no Cleanup clause for this method

Reply:
That is ok. It is always preferable to set an initial value for variable declarations. Never leave to assumptions :)